api/src/zapatos/.eslintrc.json

```diff
@@ -2,4 +2,4 @@
   "ignorePatterns": [
     "*"
   ]
-}
+}
```

Comment:
Want to keep this `No newline` here b/c it is auto-generated and we'll keep running into this issue.